.github/workflows/download-link-check-schedule.yml

```diff
@@ -0,0 +1,26 @@
+name: DVC Download Link Check
+
+on:
+  schedule:
+    - cron: '0 1-23 * * *'
```

Comment:
Wouldn't `1-23` run on only 23 hours, skipping midnight? Practically the same, I suppose, but I think `0-23` is needed for truly every hour.

Reply:
Yes, that's a typo. Thanks for spotting that!